WebCryptoAPI/sign_verify/eddsa_vectors.js

```diff
@@ -54,5 +54,20 @@ function getTestVectors() {
     vectors.push(vector);
   });
 
+  var ed25519Vector = {
+    name: "Ed25519",
+    publicKeyBuffer: new Uint8Array([9, 54, 60, 132, 200, 111, 151, 29, 166, 21, 65, 119, 70, 30, 236, 74, 150, 10, 209, 224, 115, 88, 127, 172, 52, 160, 112, 136, 220, 81, 234, 28]),
+    publicKeyFormat: "raw",
+    publicKey: null,
+    privateKeyBuffer: new Uint8Array([233, 47, 177, 106, 82, 224, 122, 208, 149, 79, 30, 251, 246, 35, 88, 194, 200, 75, 31, 47, 233, 93, 241, 52, 51, 218, 46, 210, 51, 54, 156, 4]),
+    privateKeyFormat: "raw",
```

Comment:
The specification does not allow private keys to be imported with `raw` format. The rest of the proposed changes seem to be purely stylistic.